pytest_rts/tests/test_db.py

```diff
@@ -86,3 +86,22 @@ def test_update_db_from_test_mapping():
         new_func_linenumbers = new_func_line_dict[key]
         assert old_func_linenumbers[0] + shift == new_func_linenumbers[0]
         assert old_func_linenumbers[1] + shift == new_func_linenumbers[1]
+
+
+def test_tool_files_not_mapped():
```

Comment:
This test *at the very least* needs docstring explaining what's going on. Because from the code, locally, it looks like we open some random database and check there are not things in it.

Reply:
...and not at the very least: every test should have setup phase, where it sets the things under the test. Tests should not test the state of the outside world.

Reply:
All the tests currently use the same `temp_project_repo` state. Should all tests have some individual setup phase instead of that?